src/wrappers/cobs.py

```diff
@@ -13,3 +15,21 @@ def __init__(self, index_path=COBS_INDEX_PATH):
 
     def search(self, query, threshold):
         return self.search_instance.search(query, threshold=threshold)
+
+    @staticmethod
+    def build(path_to_file_list, sample_names):
+        with TemporaryDirectory() as tmpdir:
+            doclist = cobs.DocumentList(tmpdir)
+
+            with open(path_to_file_list) as f:
+                for path in f:
+                    doclist.add(path.strip())
+
+            for doc, name in zip(doclist, sample_names):
+                doc.name = name
+
+            p = cobs.CompactIndexParameters()
+            p.term_size = COBS_TERM_SIZE
+            p.false_positive_rate = COBS_POSITIVE_RATE
+            p.clobber = True
+            cobs.compact_construct_list(doclist, COBS_INDEX_PATH, p)
```

Comment:
I think we should add the new samples into the existing dataset instead of overwriting it.